### DISCUSSION TITLE
Support building via Nix on macOS

### DISCUSSION THREAD

**Discussion by rockofox** [ghostty-org/ghostty#3823] [Feature Requests, Ideas] [answered]
It would be really cool if it was possible to build on macOS/`(aarch64|x86_64)_darwin` via Nix. Would be useful for incooperating it into e.g a [nix-darwin](https://github.com/LnL7/nix-darwin) setup.

Currently, it's [explicitly ruled out](https://github.com/ghostty-org/ghostty/blob/5293fc9c2f19b7e01472de8c2ef08bef603b754a/nix/package.nix#L200) via `package.meta.platforms`.

**Comment by 00-kat** [comment, marked as answer]
For the record, this was a duplicate of #2824.
